data/styles/clear/include/Icons.mapcss

```diff
@@ -817,6 +819,9 @@ area|z16-[tourism=picnic_site],
 node|z16-[leisure=picnic_table],
 area|z16-[leisure=picnic_table]
 {icon-image: picnic_table-m.svg;icon-min-distance: 10;}
+node|z16-[leisure=outdoor_seating],
+area|z16-[leisure=outdoor_seating],
```

Comment:
nit: just one `node|` line is enough. The second `area|` line does nothing.

(there are some tricky/buggy cases when removing a second line might break things because of peculiar inheritance behavior, but its very rare)

Reply:
Should I remove the second `area|` line from all new types that I added in this PR, then?

Reply:
yeap it'd be better!

Reply:
Removed all